deploy/osps/default/osp-rhel.yaml

```diff
@@ -450,6 +450,8 @@ spec:
               {{- end }}
               ipvsadm
 
+            systemctl disable --now firewalld || true
```

Comment:
Can you please remove the firewall configuration from here as well
https://github.com/kubermatic/operating-system-manager/blob/05da5d6be993afe0eb8e131551fbbb9fb8466c58/deploy/osps/default/osp-rhel.yaml#L427

Reply:
:+1: